compiler/src/iree/compiler/API/Internal/CompilerDriver.cpp

```diff
@@ -516,6 +586,8 @@ Error *Output::openMembuffer() {
   // Fallback to an std::string based accumulator if no platform support
   // for memfiles.
   type = Type::Membuffer;
+  // Avoid some initial memcpys with a size appropriate for program output.
+  outputString.reserve(16384);
```

Comment:
Nit: might want to define a constant at the beginning of the file for this magic number?

Reply:
I'll do so in a follow-up. There's another place in the compiler I need this, so it will be moving to a utility header soon.